### PR TITLE
[test] Add/Update test cases for RESIZE_BILINEAR op with align_corners parameter.

### PR DESCRIPTION
**Pull request by BruceDai**
@ibelem PTAL, Thanks

This PR is to
1. Updated existed test cases for RESIZE_BILINEAR op
2. Add 7 Unit test cases and 20 CTS test cases for RESIZE_BILINEAR op with align_corners parameter.

### CODE REVIEW

File: test/cts/test/V1_0/resize_bilinear.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ describe('CTS', function() {
   const assert = chai.assert;
   const nn = navigator.ml.getNeuralNetworkContext();
 
-  it('check result for Resize bilinear example', async function() {
+  it('check result for Resize bilinear with inputs(without align_corners) example', async function() {
```
Comment (ibelem) [inline test/cts/test/V1_0/resize_bilinear.js]: Tiny but please add a space between words "inputs" and "(". Same for others.

Comment (BruceDai) [reply test/cts/test/V1_0/resize_bilinear.js]: @ibelem The purpose of 'inputs(without align_corners)' text is clear to qualify 'inputs', so l skip that space. :)

**Comment by BruceDai** [comment]
@ibelem I have updated those purposes(descriptions) of test cases by adding space, please review again, thanks.

**Comment by ibelem** [comment]
LGTM, thanks!

**Pull request merged**